.yarn/versions/c07043ca.yml

```diff
@@ -0,0 +1,38 @@
+releases:
+  "@yarnpkg/cli": patch
+  "@yarnpkg/fslib": patch
+  "@yarnpkg/plugin-pnp": patch
```

Comment:
`@yarnpkg/pnp` and `@yarnpkg/core` need a release too since the hook and the zip worker got updated.